group/assignment6/tests.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from move_cars import *
+
+class TestMoveCars(unittest.TestCase):
+
+    def test_0(self):
```

Comment:
Ooh, neat corner cases!

The names and split of functions are confusing though.
If test_0 is supposed to be test_incorrect, why are the solvable cases there?
Maybe move them to test_1, add a few more wrong input cases (empty arrays, different array sizes, several of the same cars)

Or just rename the test functions. I might have misunderstood you completely:)

Reply:
You are right, I'll reorganize test functions later.